include/aws/io/io.h

```diff
@@ -240,6 +240,8 @@ enum aws_io_errors {
 
     AWS_ERROR_IO_PINNED_EVENT_LOOP_MISMATCH,
 
+    AWS_ERROR_PKCS11_ENCODING_ERROR,
```

Comment:
You should also add an entry in [io.c](https://github.com/awslabs/aws-c-io/blob/be2952e5ea2048cba35cd07e521ceb78f7587114/source/io.c#L280) with plain text for the error